src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -157,16 +154,8 @@ export default class MultipleProjectSelector extends React.PureComponent {
         onMultiSelect={this.handleMultiSelect}
         rootClassName={rootContainerStyles}
         menuFooter={({actions}) =>
-          showSubmitButton && (
-            <SubmitButtonContainer>
-              <SubmitButton
-                onClick={() => this.handleUpdate(actions)}
-                size="xsmall"
-                priority="primary"
-              >
-                {t('Apply')}
-              </SubmitButton>
-            </SubmitButtonContainer>
+          this.state.hasChanges && (
```

Comment:
We've been trying use the "show and disable instead of hide" pattern more, but I think an argument can be made here that it adds to the clutter since you can only apply if you've changed something. Thoughts here?

Reply:
The disabling thing is helpful for users who can't find some kind of functionality because they don't have permissions, and therefore assume it doesn't exist and/or is broken. I don't think it need to apply to every case universally—only when that user story seems plausible. I actually like adding the submit button here, not only because it reduces clutter, but because the transition draws the eye.